src/Core/Localization/CLDR/LocaleData.php

```diff
@@ -368,4 +382,24 @@ public function setCurrencies($currencies)
 
         return $this;
     }
+
+    /**
+     * @return TerritoryData[]
+     */
+    public function getTerritories()
```

Comment:
```suggestion
    public function getTerritories(): array
```
opr `?array` if you keep the null

Reply:
Fixed